<launch>
	<node pkg="spac2_0" exec="spac_node" name="DriveModelNode">
		<param from="$(find-pkg-share spac2_0)/config/demo_params.yaml"/>
	</node>
</launch>